README.md

```diff
@@ -1,4 +1,4 @@
-# Cocktail ![Cocktail](./logo_with_border.png)
```

Comment:
😭

Reply:
I didn't take any time at all to check how to force import this into the assets folder. It breaks in the ExDoc compiled view, so I just took it out. Feel free to push a commit :-P

Reply:
lol, no, this is fine 😅